.chloggen/update-GetStatusFromError-code.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

```suggestion
change_type: breaking
```
Should this be a breaking change since we're changing the status code that's returned?